nibabel/tests/test_scripts.py

```diff
@@ -65,6 +65,58 @@ def check_nib_ls_example4d(opts=[], hdrs_str="", other_str=""):
     assert_equal(fname, stdout[:len(fname)])
     assert_re_in(expected_re, stdout[len(fname):])
 
+
+def check_nib_diff_examples(opts=[], hdrs_str="", other_str=""):
+    # test nib-diff script
+    fnames = [pjoin(DATA_PATH, f)
+              for f in ('example4d.nii.gz', 'standard.nii.gz')]
+    fnames2 = [pjoin(DATA_PATH, f)
+              for f in ('example4d.nii.gz', 'example4d.nii.gz')]
+    code, stdout, stderr = run_command(['nib-diff'] + fnames, check_code=False)
+    assert_equal(stdout, "Field      " + pjoin(DATA_PATH, 'example4d.nii.gz') + pjoin(DATA_PATH, 'standard.nii.gz')
```

Comment:
since paths might be super long, as I have emailed, what if you print only filenames if they differ?